libraries/AP_Scripting/examples/qnh_alt.lua

```diff
@@ -0,0 +1,51 @@
+--[[
+    example that sends QNH altitude to GCS as a NAMED_VALUE_FLOAT QNG_ALT_FT in feet
```

Comment:
why feet? maybe explain that since we usually use SI for everything.

Reply:
because this is for regulatory compliance, and aviation altitudes are given in feet